xonsh/tools.py

```diff
@@ -1560,8 +1560,18 @@ def __init__(self):
         self._alias_checksum = None
         self._path_mtime = -1
 
+    def __get_possible_names(self, key):
```

Comment:
Please don't start attrs or methods with double underscore because then they are name mangled by Python and live only on the class